corehq/apps/reports/standard/project_health.py

```diff
@@ -209,23 +211,58 @@ def show_in_navigation(cls, domain=None, project=None, user=None):
     def decorator_dispatcher(self, request, *args, **kwargs):
         super(ProjectHealthDashboard, self).decorator_dispatcher(request, *args, **kwargs)
 
-    def get_filtered_group_ids(self):
-        return self.request.GET.getlist('group')
+    def get_group_location_ids(self):
+        params = self.request.GET.getlist('grouplocationfilter')
+        if params == [u'']:
+            params = []
```

Comment:
what's this working around? might be a little more readable as `params = filter(None, params)` if it's just removing empty values